_posts/2017-03-30-storage-formats-quickstep.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Storage Formats in Quickstep"
+date: 2017-03-30 14:00:30 -0600
+categories: guides
+author: Harshad
+short: Quickstep offers variety of storage formats. This post explains what these formats are and how to use them.
+---
+One of the strengths of Quickstep is the variety of storage formats it offers to store the relational data. The storage management in Quickstep can work with all these formats, and each format comes with its own strengths and weaknesses. The foundation of this work was laid in a [research paper](http://www.vldb.org/pvldb/vol6/p1474-chasseur.pdf) from the Patel Wisconsin Database group that appeared in [VLDB 2014](http://vldb.org/) (International Conference on Very Large Databases). In this post, I will provide a brief primer on what these storage formats mean and how to use them in the Quickstep system. I will also provide links to the relevant source code files, so that the readers can start exploring the code base.
```

Comment:
Just a cool markdown trick. You can use reference style links if you do not want your text to be cluttered with links: [see](https://daringfireball.net/projects/markdown/syntax#link)